src/components/steps.js

```diff
@@ -26,5 +29,30 @@ export const getSteps = (...args) => {
         new PageReviewConfiguration(...args),
         new PageProgress(...args),
     ];
-    return stepsOrder;
+
+    /* Screens can be hidden in two ways:
+     * 1. Dynamically: Controlled by the 'Page.isHidden' method in individual components,
+     *    e.g., see 'MountPointMapping.jsx'.
+     * 2. Statically: Configured via the 'hidden_screens' key in the 'anaconda.conf'
+     *    For example, the 'Account Creation' screen is hidden in the 'Workstation' ISO
+     *    because this step is handled by the 'Gnome Initial Setup' tool during the first boot.
+     */
+    return stepsOrder
+            .filter(s => {
+                const isHidden = hiddenScreens.includes(s.id);
+
+                if (isHidden) {
+                    debug(
```

Comment:
This goes only to the browser console, right ? Not relevant for this PR, but I'm thinking we might want to maybe synchronize the explicit log messages from the Web UI to the Journal as well, via cockpit bridge in the future ? We use this quite heavily with the GTK UI, where we can see user visiting screens before a crash for example - or the reasons for a screen being hidden. Without this, debugging more complex Web UI issues where we only have the log dump available from a user/customer, might be hard to impossible. :P